### PR TITLE
Feature add stripe

### PR DESCRIPTION
**Pull request by MelloFlo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ gem 'puma', '~> 3.11'
 gem 'rails', '~> 5.2.2'
 gem 'sass-rails', '~> 5.0'
 gem 'sidekiq', '~> 5.2', '>= 5.2.5'
+gem 'stripe'
```
Comment (azuryue) [inline Gemfile]: gem 'stripe', '~> 4.7'

File: app/controllers/charges_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    customer = Stripe::Customer.create(
+      email: params[:stripeEmail],
+      source: params[:stripeToken]
+    )
+
+    _charge = Stripe::Charge.create(
+      customer: customer.id,
+      amount: @amount,
+      description: 'Rails Stripe customer',
+      currency: 'usd'
```
Comment (azuryue) [inline app/controllers/charges_controller.rb]: 'euro' not 'usd'

File: app/views/charges/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<%= form_tag charges_path do %>
+  <article>
+    <% if flash[:error].present? %>
+      <div id="error_explanation">
+        <p><%= flash[:error] %></p>
+      </div>
+    <% end %>
+    <label class="amount">
+      <span>Amount: $5.00</span>
```
Comment (azuryue) [inline app/views/charges/new.html.erb]: 5.00€

**Review by fastndcurious — CHANGES_REQUESTED** [review]

File: app/controllers/charges_controller.rb (hunk 2 above)
Comment (Crysicia) [reply app/controllers/charges_controller.rb]: eur*

**Review by fastndcurious — APPROVED** [review]

**Comment by azuryue** [comment]
ignore rubocop just for an extra white-space

**Pull request merged**